packages/uiweb/src/lib/components/chat/ChatProfile/ChatProfile.tsx

```diff
@@ -166,6 +166,7 @@ export const ChatProfile: React.FC<IChatProfile> = ({
                 profile.abbrRecipient = getAbbreiatedRecipient(recipient);
                 profile.desc = profileInfo.profile?.desc;
                 profile.isGroup = false;
+                profile.web3Name = chatId.includes('.') ? chatId : null;
```

Comment:
@mishramonalisha76 I meant using the function everywhere so we don't have to use .includes everywhere and keep this logic in a single function.

Reply:
understood

Reply:
@rohitmalhotra1420  this logic is only in two places , most of the places we have to check if  chatID.includes('.')  then it doesnot something or something else. Should we just make a function to check if a string includes another substring?